### PR TITLE
Add norms

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

**Comment by JakeRoggenbuck** [comment]
I just want to do this before the project gains more traction, because we might see others contributing soon

**Comment by JakeRoggenbuck** [comment]
and thats really exciting

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+## Bug Reports & Feature Requests
 Bug reports and feature requests are greatly appreciated and can be done in form of an [issue](https://github.com/JakeRoggenbuck/auto-clock-speed/issues)
 
-# Develop
+## Develop
 If you would like to help complete a task, comment in the issue and we may assign it to you!
+
+## Pull Requests
+- Pull requests must be tested but with `cargo test` and ran on local machine (laptop with battery)
+- Pull requests that change core functionality must have a review
```
Comment (Shuzhengz) [inline CONTRIBUTING.md:11]: Maybe change that to all PRs must have at least 1 review unless it's by jake cuz jakes cool

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+
+## Bug Reports & Feature Requests
 Bug reports and feature requests are greatly appreciated and can be done in form of an [issue](https://github.com/JakeRoggenbuck/auto-clock-speed/issues)
 
-# Develop
+## Develop
 If you would like to help complete a task, comment in the issue and we may assign it to you!
+
+## Pull Requests
+- Pull requests must be tested but with `cargo test` and ran on local machine (laptop with battery)
```
Comment (Shuzhengz) [inline CONTRIBUTING.md:10]: Maybe add that WIP PRs should be drafts

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+## Milestones & Version
+- Milestones are how we prepare for a release and version number increment
+- We will add 10-20 issues to a milestone to be worked on
+- Issue move around milestones as we change our requirements and priorities
+- Bugs always get added to the closet milestone
+
+## Releases
+- Before a release, we test on several computers making sure they all function, and all function the same
+- We add a git tag to the branch and push it
+- Then we draft and publish the release
```
Comment (Shuzhengz) [inline CONTRIBUTING.md:29]: Maybe say roughly how long will the program be updated and new version released (ie every few months, every few weeks, every minute)

Comment (Shuzhengz) [inline CONTRIBUTING.md:29]: Also maybe we should start doing pre releases when we get enough attention to update on progress

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+- Pull requests must be tested but with `cargo test` and ran on local machine (laptop with battery)
+- Pull requests that change core functionality must have a review
+
+## Issues marked with `help wanted` or `good first issue`
+- Issues marked with either `help wanted` or `good first issue` are a great place to start!
+- Issues marked with these are a great place for discussion and user suggestions
+
+## Projects tab
+- The [projects tab](https://github.com/JakeRoggenbuck/auto-clock-speed/projects/1) is something we use extensively to organize when things get done
+
```
Comment (Shuzhengz) [inline CONTRIBUTING.md:19]: Just an idea, but maybe we should start thinking of a wiki to put all the documentations into?

**Review by Shuzhengz — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (JakeRoggenbuck) [reply CONTRIBUTING.md:29]: Yea, we should definitely do that, then we can test more and let people test before we publish

File: CONTRIBUTING.md (hunk 4 above)
Comment (JakeRoggenbuck) [reply CONTRIBUTING.md:19]: yee

**Comment by JakeRoggenbuck** [comment]
fixed stuff, thanks for the feedback!

**Comment by Shuzhengz** [comment]
yee!

**Review by Shuzhengz — APPROVED** [review]

**Pull request merged**